test/tests/expected/global_deps_exports_slim.js

```diff
@@ -0,0 +1,9 @@
+[
+    global,
+    function (stealRequire, stealExports) {
+        stealRequire('foo');
+        stealRequire('bar');
+        var GLOBAL = 'I don\'t like "Quotes"';
+        stealExports['GLOBAL'] = GLOBAL;
```

Comment:
@matthewp I'm not sure now if this is the way to go... Should we use `stealModule.exports = GLOBAL` for single exports? with this version you won't be able to do `var jquery = require("jquery");` but `var jquery = require("jquery").jquery;` :|